homework-submission/todo_react/src/ItemCreate.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import Delete from "./Delete";
+import Edit from "./Edit";
+import DatePicker from 'react-date-picker';
+// import DateP from "./DatePicker";
+
+export default class ItemCreate extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            to_do: [],
+            description: [],
+            deadline: [new Date()]
+        };
+    }
+
+    handleCreate = (e) => {
+        let newState = this.state
+        newState.to_do.push(this.state.description)
+        this.setState(newState)
+        //clean input
+        this.input.value = ''
+        e.preventDefault()
```

Comment:
For ordering Lines for other developers you may prevent the defaults in the first line always.